sympy/ntheory/factor_.py

```diff
@@ -2040,3 +2040,85 @@ def eval(cls, n):
                 raise ValueError("n must be a positive integer")
             else:
                 return sum(factorint(n).values())
+
+
+MERSENNE_PRIME_EXPONENTS = (2, 3, 5, 7, 13, 17, 19, 31, 61, 89, 107, 127, 521, 607, 1279, 2203,
+ 2281, 3217, 4253, 4423, 9689, 9941, 11213, 19937, 21701, 23209, 44497, 86243, 110503, 132049,
+ 216091, 756839, 859433, 1257787, 1398269, 2976221, 3021377, 6972593, 13466917, 20996011, 24036583,
+ 25964951, 30402457, 32582657, 37156667, 42643801, 43112609, 57885161, 74207281, 77232917, 82589933)
+
+def mersenne_prime_exponent(nth):
+    """
+    It returns the nth Mersenne Prime Exponents.
+    Examples
```

Comment:
Start this in a new line.

Reply:
Comments should be added from a new line?

Reply:
There should be a blank line above Examples